cornichon-docs/src/main/tut/dsl.md

```diff
@@ -274,6 +274,85 @@ And assert body.ignoring("city", "publisher").is(
 ```
 
 
+## Kafka Support (experimental)
+
+Cornichon offers a support for [Kafka](https://kafka.apache.org) v1.0.0
+
+Due to the architecture of kafka and and the handling of consumers offsets, the default execution of
```

Comment:
double `and`

Reply:
https://www.youtube.com/watch?v=Hy8kmNEo1i8

Reply:
ski-ba-bop-ba-cor-ni-chon